parm/atm/berror/hybvar_gsibec.yaml.j2

```diff
@@ -5,12 +5,12 @@ components:
     saber central block:
       saber block name: gsi covariance
       read:
-        gsi akbk: &akbk $(DATA)/fv3jedi/akbk.nc4
-        gsi error covariance file: &gsiberr $(DATA)/berror/gsi-coeffs-gfs-global.nc4
-#       gsi error covariance file: &gsiberr $(DATA)/berror/global_berror.f77
-        gsi berror namelist file: &gsibnml $(DATA)/berror/gfs_gsi_global.nml
-        processor layout x direction: &layout_gsib_x $(layout_gsib_x)
-        processor layout y direction: &layout_gsib_y $(layout_gsib_y)
+        gsi akbk: &akbk '{{ DATA }}/fv3jedi/akbk.nc4'
+        gsi error covariance file: &gsiberr '{{ DATA }}/berror/gsi-coeffs-gfs-global.nc4'
+#       gsi error covariance file: &gsiberr '{{ DATA }}/berror/global_berror.f77'
+        gsi berror namelist file: &gsibnml '{{ DATA }}/berror/gfs_gsi_global.nml'
+        processor layout x direction: &layout_gsib_x '{{ layout_gsib_x }}'
+        processor layout y direction: &layout_gsib_y '{{ layout_gsib_y }}'
```

Comment:
These should also not be quoted as that makes them strings.

Reply:
Quotes removed at e3cdbcf